src/styles/app.css

```
@@ -0,0 +1,7 @@
+body{
+    margin: 0;
+    padding: 0;
+    background: rgb(83,184,204);
+    background: linear-gradient(90deg, rgba(83,184,204,0.7511379551820728) 26%, rgba(18,208,155,0.6530987394957983) 42%, rgba(237,149,128,0.7847514005602241) 81%);
```

```suggestion
    background: linear-gradient(90deg, rgba(83,184,204,0.7511379551820728) 26%, rgba(18,208,155,0.6530987394957983) 42%, rgba(237,149,128,0.7847514005602241) 81%);
```

You don't need the above background.